ClimateEmergency_Engine/Compute/LCA/AcidificationPotential.cs

```diff
@@ -53,19 +54,19 @@ public static double AcidificationPotential(BHoMObject obj, CustomObject EPDData
                 return 0;
             }
 
-            if (EPDData.CustomData.ContainsKey("Density"))
+            if (epdData.CustomData.ContainsKey("Density"))
             {
-                density = (double)EPDData.CustomData["Density"];
+                density = (double)epdData.CustomData["Density"];
             }
             else
             {
                 BH.Engine.Reflection.Compute.RecordError("The EPDDataset must have a valid value for density under a 'Density' key.");
                 return 0;
             }
```

Comment:
@FraserGreenroyd I can use ```try catch``` with proper convert cast for all BHoM obj querying methods if needed. What do you recommend?

Reply:
I'm happy with a casting in this manner, but if you'd rather use `System.Convert.ToDouble` then I wouldn't be against it as it is slightly more obvious then what it's doing (casting may not be obvious to the novice coming into BHoM) :smile: